src/pages/individual/Individual.jsx

```diff
@@ -180,6 +180,8 @@ export default function Individual() {
     setError: setPatchError,
   } = usePatchIndividual();
 
+  const error = deleteError || patchError;
```

Comment:
as discussed in slack, i think this should be removed

Reply:
Resolved by merge with develop